SUPPORT.rst

```diff
@@ -11,7 +11,7 @@ There are several ways to contact us or to get help:
    `\ **Discourse Forums** <https://discourse.mozilla.org/c/deep-speech>`_ - If your question is not addressed in the `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_\ , the `Discourse Forums <https://discourse.mozilla.org/c/deep-speech>`_ is the next place to look. They contain conversations on `General Topics <https://discourse.mozilla.org/t/general-topics/21075>`_\ , `Using Deep Speech <https://discourse.mozilla.org/t/using-deep-speech/21076/4>`_\ , and `Deep Speech Development <https://discourse.mozilla.org/t/deep-speech-development/21077>`_.
 
 #. 
-   `\ **IRC** <https://wiki.mozilla.org/IRC>`_ - If your question is not addressed by either the `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ or `Discourse Forums <https://discourse.mozilla.org/c/deep-speech>`_\ , you can contact us on the ``#machinelearning`` channel on `Mozilla IRC <https://wiki.mozilla.org/IRC>`_\ ; people there can try to answer/help
+   `\ **Matrix chat** <https://matrix.to/#/!VkKaHlfgpCUnNSLuWd:mozilla.org?via=mozilla.org>`_ - If your question is not addressed by either the `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ or `Discourse Forums <https://discourse.mozilla.org/c/deep-speech>`_\ , you can contact us on the ``#machinelearning`` channel on `Mozilla Matrix <https://matrix.to/#/!VkKaHlfgpCUnNSLuWd:mozilla.org?via=mozilla.org>`_\ ; people there can try to answer/help
```

Comment:
Just checked this link and I think it doesn't work. But this link
```
https://chat.mozilla.org/#/room/!USFCphMyGNrPSFlRph:mozilla.org
```
seems to work.

Reply:
It works for me, it's just a bit convoluted. But I'll revert to chat.mozilla.org.